src/explore-education-statistics-common/src/modules/table-tool/utils/__tests__/getDefaultTableHeadersConfig.test.ts

```diff
@@ -380,74 +325,59 @@ describe('getDefaultTableHeadersConfig', () => {
   });
 
   test('returns correct config with siblingless filters removed when one option for every filter type', () => {
-    const testTableData: TableDataResponse = {
-      subjectMeta: {
-        filters: {
-          Characteristic: {
-            totalValue: '',
-            hint: '',
-            legend: 'Characteristic',
-            name: 'characteristic',
-            options: {
-              EthnicGroupMajor: {
-                label: 'Ethnic group major',
-                options: [
-                  {
-                    label: 'Ethnicity Major Black Total',
-                    value: 'ethnicity-major-black-total',
-                  },
-                ],
-              },
-            },
-          },
-          SchoolType: {
-            totalValue: '',
-            hint: 'Filter by school type',
-            legend: 'School type',
-            name: 'school_type',
-            options: {
-              Default: {
-                label: 'Default',
-                options: [
-                  {
-                    label: 'State-funded secondary',
-                    value: 'state-funded-secondary',
-                  },
-                ],
-              },
-            },
-          },
-        },
-        footnotes: [],
-        geoJsonAvailable: false,
-        indicators: [
+    const testTableDataSiblingless = JSON.parse(JSON.stringify(testTableData));
```

Comment:
I think for cases like this, can you achieve the same behaviour without the JSON-jugging by using object spreading like:

```
const testTableDataSiblingless = {
    ...testTableData,
    subjectMeta.filters.Characteristic.options: {
      EthnicGroupMajor: {
        blah blah
      }
    }
};
```

?

This should create the deep copy that you're after whilst also letting you override the bits that you want to have specifically different.